enterprise/frontend/src/metabase-enterprise/table-editing/table-edit/toasts/getUpdateErrorMessage.tsx

```diff
@@ -15,11 +15,11 @@ export const getUpdateApiErrorMessage = (
     Array.isArray(maybeError.data?.errors) &&
     "error" in maybeError.data?.errors[0]
```

Comment:
I suppose you also want to check for `"message"` in `maybeError.data?.errors[0]`